Frontend initialization

mechanicjo opened the pull request:
간단하게 버튼들이랑 navigation 이용하여 페이지 전환 몇 개 구현해봤습니다.
-Login page
-Signup page
-Home page
-Search page
-My page

Commit:
INIT: create new android project

Commit:
FRONTEND: frontend draft

eldpswp99 commented on frontend/app/src/main/java/com/team13/fooriend/MainActivity.kt:
```diff
@@ -0,0 +1,224 @@
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun LogInPage(navController: NavController){
```
현재 이 페이지들은 간이로 추가된거고, 나중에 따로 파일로 분리되는건가요?

mechanicjo replied:
네 추후에 페이지 별로나 큰 프로세스 별로 나눠서 파일 분리할 예정입니다.

Commit:
gitignore에 .idea 추가 및 .getkeep 삭제

Commit:
.idea 데이터 삭제

ozeeeno committed:
deployment : Mypage first detailing

ozeeeno committed:
Revert "deployment : Mypage first detailing"

This reverts commit 31f9f17e6fd512ef7664a8a093f218534cf6cadc.

Commit:
Revert "Revert "deployment : Mypage first detailing""

This reverts commit 9d7e398822da41bc42e3571eff08f2c14b3ff515.

Commit:
파일 분리

Commit:
Rename: 파일들이나 fun 이름을 convention에 맞춰서 page -> screen 으로 수정, 조교님 피드백을 반영하여  search -> social로 수정

mechanicjo committed:
Add Feature: BottomBar

navGraph와 Scaffold를 이용하여 BottomBar를 만들어 보았습니다.
하단에 home, social, mypage에서 선택하여 screen을 전환할 수 있습니다.

mechanicjo committed:
각 screen들의 preview 추가

개발의 편의성을 위하여 각 screen을 컴파일 하지 않고도 android studio에서 볼 수 있도록 preview를 추가하였습니다.

mechanicjo committed:
navigation 계층화

root navigation을 만들어서 navigation을 계층화 했습니다.
그런데 Navigation Bar에서 터지네요..

mechanicjo committed:
Merge pull request #6 from snuhcs-course/frontend_navigation

navigation 계층화

mechanicjo committed:
Error Correction: 이제 login과 bottomBar가 모두 잘 작동됩니다.

jakehsj reviewed (approved):
navigation bar 문제점이 잘 수정된것 같습니다

ozeeeno committed:
build: homescreen map implementation

mechanicjo committed:
feat(map): 홈 스크린에 지도 뜨게 설정
